package.json

```diff
@@ -34,5 +34,8 @@
 		"cucumberJson": {
 			"generate": true
 		}
+	},
+	"dependencies": {
+		"cypress-file-upload": "^5.0.8"
```

Comment:
this dependencies should be inside -> "devDependencies" in line # 26 in this file